src/main.rs

```diff
@@ -32,12 +37,17 @@ struct Args {
 
 fn main() {
     let args = Args::parse();
-    let bytecode = args.bytecode;
 
-    let bytecode = if let Some(stripped) = bytecode.strip_prefix("0x") {
-        stripped.to_string()
-    } else {
+    let bytecode = if let Some(bc) = args.bytecode {
+        bc
+    } else if let Some(file) = args.bytecode_file {
+        let mut file = File::open(file).unwrap();
+        let mut bytecode = String::new();
+        file.read_to_string(&mut bytecode).unwrap();
         bytecode
```

Comment:
This doesn't work as it may read the char return at the end of the file. A workaround would be to only read the first line:

```rust
bytecode
    .lines()
    .next()
    .expect("Should provide a one-liner content")
    .to_string()
```

Reply:
Tested with a few more lines, seems to work unless there are some illegal characters, lmk if I still need to do this

Reply:
Good solution !